
try:
    2/0
except:
    print("Houve algum erro")
finally:
    print("O programa foi finalizado")